ngraph/frontend/onnx_import/src/onnx_import/op/round.cpp

```diff
@@ -30,10 +30,36 @@ namespace ngraph
         {
             namespace set_1
             {
+                // WARNING!
```

Comment:
I think that `Round(x) = Floor(x + 0.5 * Sign(x))`. It is a bit easier to implement and should work faster (because all the operations are elementwise and should be effectively fused in the plugin) than what you wrote below. It also works for negative numbers. Please, correct me if I am wrong.

Reply:
@lazarevevgeny 
Let's consider `data=-1.1 -> Floor(-1.1 + 0.5*(-1)) = Floor(-1.6) = -2`
while for current version:
`data_floor = Floor(-1.1) = -2`
`diff = data - data_floor = -1.1 - (-2) = 0.9`
because (diff < 0.5) != true
`round = -2 + 1 = -1`
So I am not sure about negative numbers. Do I miss something?

Reply:
Ok. I see the problem in my formula. How about this one: `Round(x) = Floor(x + 0.5)`?

Reply:
@lazarevevgeny Good point, it works correctly for all cases supported by current version. Thanks